### DISCUSSION TITLE
How to delete particles

### DISCUSSION THREAD

**Discussion by happtain** [InteractiveComputerGraphics/SPlisHSPlasH#335] [Q&A]
Hello,
I would like to make a scene where an emitter continuously emits tens of thousands of particles, such as a fountain, but too many particles will increase the burden on the system. Is there any way to delete particles that leave the specified area during the simulation? 

Best Regard.

**Comment by janbender** [comment]
You can define a bounding box for the emitter and when a particle is outside of this box it is reused (emitted again as new particle). See here: 

https://splishsplash.readthedocs.io/en/latest/file_format.html#id2
